Samples/1.Logon/Program.cs

```diff
@@ -76,14 +76,6 @@ static void Main( string[] args )
 
         static void OnConnected( SteamClient.ConnectedCallback callback )
         {
-            if ( callback.Result != EResult.OK )
```

Comment:
Yup. We only ever triggered the callback with EResult.OK, so it’s effectively redundant.